### PR TITLE
Java bindings for ApplePayDomain

### PR DESCRIPTION
**Pull request by vijay-singh**
Bindings for ApplePayDomain

### CODE REVIEW

File: src/main/java/com/stripe/model/DeletedApplePayDomain.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.stripe.model;
+
+
```
Comment (brandur) [inline src/main/java/com/stripe/model/DeletedApplePayDomain.java]: Super nit, but mind just tightening up the whitespace here? Thanks!

File: src/main/java/com/stripe/model/ApplePayDomain.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+	public void setDomainName(String domainName) {
+		this.domainName = domainName;
+	}
+
+	public static ApplePayDomain create(Map<String, Object> params)
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return create(params, (RequestOptions) null);
+	}
```
Comment (brandur) [inline src/main/java/com/stripe/model/ApplePayDomain.java]: Mind grouping these methods together by name? (i.e. Both versions of `create` are colocated in the file.) I know it's kind of a mess elsewhere, but we should be aiming to organize these a little better.

File: src/main/java/com/stripe/model/ApplePayDomain.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+		return request(RequestMethod.DELETE, getInstanceURL(id), null, DeletedApplePayDomain.class, options);
+	}
+
+	public static ApplePayDomainCollection list(Map<String, Object> params, RequestOptions options)
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return requestCollection(getClassURL(), params, ApplePayDomainCollection.class, options);
+	}
+
+	public static String getClassURL() {
```
Comment (brandur) [inline src/main/java/com/stripe/model/ApplePayDomain.java]: I notice that most of the other models with this method don't use `static`. Would you mind making it an instance method for consistency?

Also, can you see if you can change it to private visibility? I don't think there's any reason to leak this interface out of the package.

File: src/main/java/com/stripe/model/DeletedApplePayDomain.java (hunk 1 above)
Comment (vijay-singh) [inline src/main/java/com/stripe/model/DeletedApplePayDomain.java]: done!

File: src/main/java/com/stripe/model/ApplePayDomain.java (hunk 2 above)
Comment (vijay-singh) [inline src/main/java/com/stripe/model/ApplePayDomain.java]: done!

File: src/main/java/com/stripe/model/ApplePayDomain.java (hunk 3 above)
Comment (vijay-singh) [inline src/main/java/com/stripe/model/ApplePayDomain.java]: changed to private visibility

**Comment by brandur** [comment]
Thanks! This looks good to me.

**Comment by brandur** [comment]
Released as 2.10.0.

**Pull request merged**